server/src/delta_cache.rs

```diff
@@ -0,0 +1,286 @@
+use std::collections::VecDeque;
+use unleash_types::client_features::{DeltaEvent, ClientFeature, Segment};
+
+
+
+#[derive(Debug, Clone)]
+pub struct DeltaHydrationEvent {
+    event_id: i32,
```

Comment:
Should this not be a `u32` or a `usize`? Probably not a big deal just seems weird to add scope for negative numbers when they can't be negative by design in this feature

Reply:
We did it for consistency, how other ids have been implemented in types. For example segment id, 
variant id are are i32.


https://github.com/Unleash/unleash-types-rs/blob/d782b656f6bd4887bc8251a95bbbd6b004612aeb/src/client_features.rs#L304

https://github.com/Unleash/unleash-types-rs/blob/d782b656f6bd4887bc8251a95bbbd6b004612aeb/src/client_features.rs#L266

Reply:
Fair enough but I'd still like to see this as an unsigned type when dealing with a monotonically increasing id

Won't fuss if you don't care though

Reply:
Done